lib/resources/r4/related_person.yaml

```diff
@@ -147,8 +147,9 @@ fields:
     }
   note: |
     <ul>
-      <li>The <code>relationship.coding</code> must contain exactly one proprietary system.</li>
-      <li>The <code>relationship.coding</code> cannot contain a standard system.</li>
+      <li> The <code>relationship.coding</code> must contain exactly one proprietary system. </li>
+      <li> The <code>relationship.coding</code> cannot contain a standard system. </li>
```

Comment:
Should we also not mention the same criteria for the relationship.extension for relation?
@alexbeckner1